llvm/lib/Target/RISCV/AsmParser/RISCVAsmParser.cpp

```diff
@@ -597,7 +613,10 @@ struct RISCVOperand final : public MCParsedAsmOperand {
     if (!isImm())
       return false;
     bool IsConstantImm = evaluateConstantImm(getImm(), Imm, VK);
-    if (VK == RISCVMCExpr::VK_RISCV_LO || VK == RISCVMCExpr::VK_RISCV_PCREL_LO)
+    if (VK == RISCVMCExpr::VK_RISCV_LO ||
```

Comment:
None of these change to accept more values for `VK` are tested.

Reply:
I think the tests added in RISCV/MC should cover this now. Please let me know if I've missed one of the cases.